common-docs/content/common-theme/shortcodes/objectives.md

```diff
@@ -0,0 +1,36 @@
++++
+title="Objectives"
+description="Passing objectives from GitHub to Hugo"
+emoji="🎯"
+menu=["shortcodes"]
++++
+
+```go
+<!--{{</* objectives */>}}-->
```

Comment:
Why is this double commented, both an HTML comment and a Hugo comment?

(Same question for tasks.md)

Reply:
I understand the HTML comment - I'm not sure about the Hugo one? What would happen if we just wrote:
```
<!--{{<objectives>}}-->
```

instead of
```
<!--{{</* objectives */>}}-->
```

Reply:
Oh, Hugo executes the shortcode inside the code fence if you do that.